pkgs/sdk_triage_bot/lib/src/prompts.dart

```diff
@@ -45,13 +47,110 @@ If the issue is clearly a bug report, then also apply the label 'type-bug'.
 If the issue is mostly a question,  then also apply the label 'type-question'.
 Otherwise don't apply a 'type-' label.
 
+If the issue title starts with "[breaking change]" then apply the
+`breaking-change-request` label but don't assign an area label. IMPORTANT: only
+do this if the issue title starts with "[breaking change]".
```

Comment:
Tricky. I think we actually *shouldn't* add the tag.

Users can choose to write `[breaking change]`, but that doesn't make it a breaking change request. *Real* breaking change requests come from the template, which should set labels too. That means that if the `breaking-change-request` label isn't already set, this is probably *not* a real breaking change request, just a user cargo-culting our process, and what they have is an enhancement request, likely one that does need an area.

Reply:
I think I agree. I'll change this so the bot avoids triaging `[breaking change]` issues; manual review can figure out if they actually need any action.